vault/blog.community.2021.new-user-survey.md

```diff
@@ -0,0 +1,173 @@
+- Apple Notes (4%)
+- Plain Markdown or text files (4%)
+- Trello / Confluence / Jira (4%)
+- Typora (3%)
+- Physical notes (2%)
+
+## Do users have plans on publishing their notes?
+
+![Bar graph displaying levels of interest in publishing with Dendron](https://org-dendron-public-assets.s3.amazonaws.com/images/2021-survey-publishing-interest.png)
+
+New users were only able to select a single response, if they were to respond to this question at all. A majority of users were **Curious** (39%) about publishing, or otherwise already saying **Yes** (36%) to planning publishing. The remaining respondents said **No** (25%) and were not interested in Dendron's publishing capabilities.
+
+## Takeaways
+
+Users are drawn to Dendron for Personal Knowledge Management (PKM), task management, research, and more.
+
+With that in mind, Dendron and the Dendron community have been introducing more features and enhancements that continue to develop:
+
+- [Task Notes](https://wiki.dendron.so/notes/SEASewZSteDK7ry1AshNG/)
+  - [Contribute to the GitHub Discussion](https://github.com/dendronhq/dendron/discussions/1358)
+- [Note Traits](https://wiki.dendron.so/notes/EQoaBI8A0ZcswKQC3UMpO/)
+- [The Open PKM Catalogue](https://pkm.dendron.so): This work-in-progress is meant to be a reference of all things PKM.
+- Improvements to [importing existing Markdown](https://wiki.dendron.so/notes/f23a6290-2dec-45dc-b616-c218ee53db6b/)
+- Better workflows, such as using [multiple vaults with private and public notes](https://wiki.dendron.so/notes/6DZiBObwhZNYRjnokQ422/)
+
+Though half of respondents said they were developers, the other half of new users came from tech and non-tech backgrounds. This includes many users where Dendron is their introduction to [VS Code](https://code.visualstudio.com/), [git, and GitHub](https://wiki.dendron.so/notes/vsAxYld4zsbD2XjKoh7OP/).
+
+Here are a few areas where we've taken action:
+
+- [Refreshed Getting Started Guide](https://wiki.dendron.so/notes/678c77d9-ef2c-4537-97b5-64556d6337f1/)
+- GitHub Templates for easier, more automated publishing
+  - [Publish with Netlify](https://wiki.dendron.so/notes/yetuum6o9wZi6eVJQBbQb/)
+  - [Publish with GitHub Pages and GitHub Actions](https://wiki.dendron.so/notes/FnK2ws6w1uaS1YzBUY3BR/)
+- Improved `Change Workspace` and `Initialize Workspace` UX
+
+Many of our new users are coming to Dendron as their first tool for personal knowledge management than from any other tool. Even then, the amount of different workflows and tooling that people come from (or are using in parallel) is a massive list.
+
+We have a [migration guide](https://wiki.dendron.so/notes/f9b4fc21-7613-4c8a-9257-cec4c06b67f9/) that can be referenced for people wanting to import their existing notes into Dendron. With so many tools out there, we are open to [any docs contributions and improvements](https://github.com/dendronhq/dendron-site/blob/master/vault/dendron.guides.migration.md) whenever someone transforms into a dendronite.
```

Comment:
For folks coming from existing tools,

Reply:
active style. call to action. "we are looking to the community to help us improve our migration guide"